.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
Feedback applied. I'm ok with renaming the package to `mocha-ui-skipper`.

The idea behind skipping tests with this UI is that it allows us to skips certain tests that are inherited from different modules (like `cloudant` inheriting `couchdb2` but skipping 1/2 test cases). This way we don't have to modify test cases to skip certain ones based on a flag. 

In a project using this we would ideally set the Environment variable in `/test/init.js` like done so in `couchdb2`. So CI should still have all tests passing except the ones explicitly stated in `init.js` ... similar to feature flags used in juggler but this is just cleaner / easier. 

Example from `couchdb2`: 

```
var skips = [
  'find all limt ten',
  'find all skip ten limit ten',
  'find all skip two hundred',
  'isActual',
];

if (process.env.LOOPBACK_MOCHA_SKIPS) {
  process.env.LOOPBACK_MOCHA_SKIPS =
    process.env.LOOPBACK_MOCHA_SKIPS.concat(skips);
} else {
  process.env.LOOPBACK_MOCHA_SKIPS = skips;
}
```